html_css/css/style.css

```diff
@@ -1027,47 +1124,55 @@ button:active {
   cursor: pointer;
 }
 
-.blog__container_scroll-btn:hover {
+.blog__slider_scroll-btn:hover {
   box-shadow: 0px 4px 16px var(--arrow-box-shadow);
 }
 
-.blog__container_scroll-btn:active {
+.blog__slider_scroll-btn:active {
   border: 1px solid var(--arrow-play-button-hover);
 }
 
-.form {
+.registration-form {
+  padding: 78px 302px 0px 302px;
+}
+
+.registration-form form {
```

Comment:
Looks like wrong class name. It couldn`t be registration-form if form already nested in it. It could be form-container, for example

Reply:
changed